v0/destinations/wootric/util.js

```diff
@@ -86,12 +102,12 @@ const retrieveUserId = async (userId, accessToken) => {
   const processedUserResponse = processAxiosResponse(userResponse);
 
   if (processedUserResponse.status === 200) {
-    return processedUserResponse.response?.id;
+    return processedUserResponse.response;
   }
 
   if (processedUserResponse.status !== 404) {
     throw new CustomError(
-      `[Wootric]:: Unable to retrieve userid due to ${JSON.stringify(
+      `[Wootric]:: Unable to retrieve userId due to ${JSON.stringify(
```

Comment:
In 404 situation we are not handing the case outside

Reply:
updated